### DISCUSSION TITLE
M600 won't update

### DISCUSSION THREAD

**Discussion by jfmmm** [NilsRo/OctoPrint-SlicerEstimator#58] [Q&A] [answered]
I can see the times to the next filament change as shown in the screenshot but the time won't update unless I refresh the page.

Is that normal?

**Comment by NilsRo** [comment]
> I can see the times to the next filament change as shown in the screenshot but the time won't update unless I refresh the page.
> 
> 
> 
> Is that normal?

Hi. 

Could you share Browser, OctoPrint and Plugin Version and at best a screenshot. From your description it should be a bug, will test it in parallel. In the past this worked fine.

Regards 
Nils

**Comment by NilsRo** [comment, marked as answer]
I corrected an issue with the update in the GUI in version 1.6.1
You can try if it is working fine there now.  To get it in Octoprint immediately you have to force the check for new versions.
